File: src/Empleado.java
public class Empleado extends Persona {
    public String rol;

    public Empleado(String nombre, String apellido, String direccion, String rol) {
        super(nombre, apellido, direccion);
        this.rol = rol;
    }

    public String mostrarInfoConRol() {
        return super.mostrarInfo() + "rol: " + rol;
    }
}

File: src/Empleado.py
from Persona import Persona

class Empleado(Persona):
    def __init__(self, nombre, apellido, direccion, rol):
        super().__init__(nombre, apellido, direccion)
        self.rol = rol

    def mostrar_info_con_rol(self):
        return f"{super().mostrar_info()}, rol: {self.rol}"
